sdk/ruby/README.md

```diff
@@ -2,26 +2,21 @@
 
 ## Usage
 
-### Installing the library
+### Get the gem
 
-#### Via Rubygems
+The Ruby SDK is available [via Rubygems](https://rubygems.org/gems/chain-sdk). Make sure to use the most recent version whose major and minor components (`major.minor.x`) match your version of Chain Core. Ruby 2 is required.
 
-TBA
-
-#### Via downloaded .gem
-
-Install the gem into your gem library:
+For most applications, you can simply add the following to your `Gemfile`:
 
 ```
-gem install --local chain-sdk-<VERSION>.gem
+gem 'chain-sdk', '~> 1.0.0', require: 'chain'
```

Comment:
I think pessimistic `1.0.0` works with our versioning. They'll just get Ruby patch updates; let's keep this until we see a reason not to?